clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/ops/servergroup/AbstractEnableDisableKubernetesAtomicOperation.groovy

```diff
@@ -102,6 +102,11 @@ abstract class AbstractEnableDisableKubernetesAtomicOperation implements AtomicO
       }
     }
 
+    if (!pods || pods.size() == 0) {
```

Comment:
I think `!pods` is sufficient here since it'll return `true` if it's `null` or empty.